.travis.yml

```diff
@@ -10,4 +11,4 @@ after_success:
   # push coverage results to Coveralls
   - julia -e 'cd(Pkg.dir("WebIO")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
   # push coverage results to Codecov
-  - julia -e 'cd(Pkg.dir("WebIO")); Pkg.add("Coverage"); using Coverage; Codecov.submit(Codecov.process_folder())'
```

Comment:
This looks odd -- why do we cd to WebIO.jl?

I don' think you changed this during this PR, but I just noticed it...